addons/mail/static/src/core/common/message_model.js

```diff
@@ -40,6 +49,23 @@ export class Message extends Record {
     attachments = Record.many("Attachment", { inverse: "message" });
     author = Record.one("Persona");
     body = Record.attr("", { html: true });
+    bodyMarkdown = Record.attr("", {
```

Comment:
Confusing field name, this is not containing `bodyMarkdown` but `bodyHtml`/`bodyMarkup` or something like that.

Reply:
I think `bodyHtml` is fine